.github/workflows/build-test.yaml

```diff
@@ -2567,21 +2571,16 @@ jobs:
             exit 1
           fi
 
-          if [ "$(kubectl get all -n postgres-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "postgres-test namespace still contains resources\n\n"
-            kubectl get all -n postgres-test
-            exit 1
-          fi
 
-          if [ "$(kubectl get all -n redis-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "redis-test namespace still contains resources\n\n"
-            kubectl get all -n redis-test
+          if [ "$(kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG" --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
+            printf "%s namespace still contains resources\n\n" "$ns"
+            kubectl get all,secrets,configmap,pvc -A -l "kots.io/app-slug=$APP_SLUG"
             exit 1
           fi
 
-          if [ "$(kubectl get all -n nginx-test --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
-            printf "nginx-test namespace still contains resources\n\n"
-            kubectl get all -n nginx-test
+          if [ "$(kubectl get crd rabbitmqclusters.rabbitmq.com --ignore-not-found | wc -l | tr -d '\n')" != "0" ]; then
+            printf "rabbitmqclusters.rabbitmq.com CRD was not removed\n\n"
+            kubectl get crd rabbitmqclusters.rabbitmq.com
```

Comment:
Checking for CRD not found is good
We don't need to check for `rabbitmqclusters` since if CRD is not present CRs won't be present
[+ hard to validate the error]
```
# kubectl get rabbitmqclusters.rabbitmq.com -n rabbitmq-system --ignore-not-found
error: the server doesn't have a resource type "rabbitmqclusters"
```